### DISCUSSION TITLE
Does it work on clamshell mode?

### DISCUSSION THREAD

**Discussion by ozmo1925** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#124] [Q&A]
I would like to use this for my m1 mba but I wonder if it works under clamshell mode and can still be  set % 80 battery?

**Comment by KissDanielMark** [comment]
https://github.com/davidwernhart/AlDente/discussions/119

**Comment by leeummm** [comment]
Yes it works in clamshell mode
